src/development/LocalTest/Controllers/HomeController.cs

```diff
@@ -68,7 +68,7 @@ public async Task<IActionResult> Index()
             model.AppPath = _localPlatformSettings.AppRepositoryBasePath;
             model.StaticTestDataPath = _localPlatformSettings.LocalTestingStaticTestDataPath;
             model.LocalAppUrl = _localPlatformSettings.LocalAppUrl;
-            var defaultAuthLevel = _localPlatformSettings.LocalAppMode == "http" ? await GetAppAuthLevel(model.TestApps.First().Value) : 2;
+            var defaultAuthLevel = _localPlatformSettings.LocalAppMode == "http" ? await GetAppAuthLevel(model.TestApps) : 2;
```

Comment:
@lorang92 Should I create a new PR for just the fixes in HomeController? It would be convenient if LocalTest in `"http"` mode would not crash if no app is running.

Reply:
Yes